public/docs/_examples/cb-item-template/e2e-spec.ts

```diff
@@ -0,0 +1,43 @@
+/// <reference path='../_protractor/e2e.d.ts' />
+'use strict';
+// FIRST RUN:       gulp run-e2e-tests --filter=cb-item-template
+// SUBSEQUENT RUNS: gulp run-e2e-tests --filter=cb-item-template --fast
+describe('Item template renderer', function () {
+
+  beforeAll(function () {
+    browser.get('');
+  });
+
+  it('should toggle the menu', function () {
+
+    expect( element( by.className( 'select-items' ) ).isPresent() ).toBe( false );
```

Comment:
All the spaces between parens parameters are nice, but that is not the convention we use in this project so I would ask you for remove them in all the files.